contact.php

```diff
@@ -0,0 +1,61 @@
+<?php include'head.php';
+require_once'bd.php';
+
+$reponse=$bdd->query("SELECT * FROM contact ORDER BY id DESC");
+
+if(isset($_POST['submit'])){
+    $name1=$_POST['fname'];
+    $name2=$_POST['lname'];
+    $email=$_POST['email'];
+    $subject=$_POST['subject'];
+    $text=$_POST['text'];
+    
+    $req=$bdd->prepare("INSERT INTO contact SET fname=?, lname=?,email=?,subject=?, comment=?");

```

Comment:
Right now, the user can input almost anything into your database. Perhaps you want to at least check whether e.g. the text has a certain length or the names are not empty.